pkg/caveats/types/map.go

```diff
@@ -7,9 +7,9 @@ import (
 )
 
 func init() {
-	registerMethodOnDefinedType(cel.MapType(cel.StringType, cel.AnyType),
+	registerMethodOnDefinedType(cel.MapType(cel.StringType, cel.DynType),
```

Comment:
Is this a breaking change? is it possible this would render existing serialized CEL AST incompatible?

Reply:
No